data/src/main/java/com/playone/mobile/data/source/BufferooDataStoreFactory.kt

```diff
@@ -16,25 +16,17 @@ open class BufferooDataStoreFactory @Inject constructor(
      * Returns a DataStore based on whether or not there is content in the cache and the cache
      * has not expired
      */
-    open fun retrieveDataStore(): BufferooDataStore {
-        if (bufferooCache.isCached() && !bufferooCache.isExpired()) {
-            return retrieveCacheDataStore()
-        }
-        return retrieveRemoteDataStore()
-    }
+    open fun retrieveDataStore() =
+        if (bufferooCache.isCached() && !bufferooCache.isExpired())  retrieveCacheDataStore() else retrieveRemoteDataStore()
```

Comment:
How about make the line be less than 100 characters? (it is too long to see whole the line)
I think it is fine to be
```
if (bufferooCache.isCached() && !bufferooCache.isExpired())  
    retrieveCacheDataStore() else retrieveRemoteDataStore()
```

what do you think?

Reply:
Sure, that's cool. 👍